doc/source/examples/similarity_explanations_mnist.ipynb

```diff
@@ -0,0 +1,599 @@
+{
```

Comment:
<p>In this notebook we apply -&gt; In this notebook, we apply</p><p> </p><p>in the train dataset -&gt; in the training dataset</p><p> </p><p>was label as a 4 -&gt; was labeled as a 4</p><p> </p><p>pytorch, tensorflow -&gt; <code>pytorch</code>,<code>tensorflow</code></p><p> </p><p>consider rephrasing: is implemented with both pytorch and tensorflow -&gt; supports pytorch and tensorflow backends?</p><p> </p><p>link not working: can be found here.</p><p> </p><p>replace papers' links with link to the abstract </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>